### PR TITLE
navidrome 0.51.1 (new formula)

### PR DESCRIPTION
**Pull request by printfn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/n/navidrome.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Navidrome < Formula
+  desc "Modern Music Server and Streamer compatible with Subsonic/Airsonic"
+  homepage "https://www.navidrome.org"
+  url "https://github.com/navidrome/navidrome.git",
+      tag:      "v0.51.1",
+      revision: "6d253225de088e27f9a191d6bcedcb2e68b123fa"
```
Comment (carlocab) [inline Formula/n/navidrome.rb:6]: Do we need a Git clone? Can we use a tarball instead?

File: Formula/n/navidrome.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+  depends_on "node@18" => :build
+  depends_on "pkg-config" => :build
+  depends_on "taglib"
```
Comment (carlocab) [inline Formula/n/navidrome.rb:12]: Don't you need this?
```suggestion
  depends_on "pkg-config" => :build
  depends_on "ffmpeg"
  depends_on "taglib"
```

Comment (printfn) [reply Formula/n/navidrome.rb]: fixed

File: Formula/n/navidrome.rb (hunk 1 above)
Comment (printfn) [reply Formula/n/navidrome.rb]: Fixed

File: Formula/n/navidrome.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+class Navidrome < Formula
+  desc "Modern Music Server and Streamer compatible with Subsonic/Airsonic"
+  homepage "https://www.navidrome.org"
+  url "https://github.com/navidrome/navidrome/archive/refs/tags/v0.51.1.tar.gz"
+  sha256 "fc962e3acbedfad63934eda016d4e380dd3a06b4636f2b1e61ade9700a2addcd"
+  license "GPL-3.0-only"
+  head "https://github.com/navidrome/navidrome.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "node@18" => :build
```
Comment (chenrui333) [inline Formula/n/navidrome.rb:10]: any issue tracker for building with latest node?

Comment (printfn) [reply Formula/n/navidrome.rb:10]: Looks like they recently merged an update to use Node 20, and it already builds fine with that version. https://github.com/navidrome/navidrome/pull/2861
Should I update this PR to use the latest node?

Comment (chenrui333) [reply Formula/n/navidrome.rb:10]: yes, please

Comment (printfn) [reply Formula/n/navidrome.rb]: fixed!

**Review by chenrui333 — APPROVED** [review]
lgtm

**Pull request merged**